addons/account/models/account_bank_statement.py

```diff
@@ -203,31 +193,20 @@ def _search_is_valid(self, operator, value):
             return [('id', 'in', invalid_ids)]
         return [('id', 'not in', invalid_ids)]
 
-    # -------------------------------------------------------------------------
-    # CRUD
-    # -------------------------------------------------------------------------
-    @api.model_create_multi
-    def create(self, vals_list):
-        # EXTENDS base
-        # If we are doing a split, we have to correct the split statement's balance to keep both original and new
-        # statements valid.
-        if self._context.get('split_line_id'):
-            old_statement = self.env['account.bank.statement.line'].browse(self._context.get('split_line_id')).statement_id
-            old_lines = old_statement.line_ids
-        statements = super().create(vals_list)
-        if self._context.get('split_line_id'):
-            statements.ensure_one()
-            if old_statement:
-                net_change = sum((statements.line_ids & old_lines).filtered(lambda l: l.state == 'posted').mapped('amount'))
-                old_statement.balance_start += net_change
-        return statements
-
     # -------------------------------------------------------------------------
     # BUSINESS METHODS
     # -------------------------------------------------------------------------
+    def _get_statement_validity(self):
+        """ Compares the balance_start to the previous statements balance_end_real """
+        self.ensure_one()
+        previous = self.env['account.bank.statement'].search([
+                ('first_line_index', '<', self.first_line_index),
+                ('journal_id', '=', self.journal_id.id)
+            ], limit=1, order='first_line_index DESC')
```

Comment:
cosmetic suggestion:
```suggestion
        previous = self.env['account.bank.statement'].search([
            ('first_line_index', '<', self.first_line_index),
            ('journal_id', '=', self.journal_id.id)
        ], limit=1, order='first_line_index DESC')
```
or
```suggestion
        previous = self.env['account.bank.statement'].search(
            [
                ('first_line_index', '<', self.first_line_index),
                ('journal_id', '=', self.journal_id.id)
            ],
            limit=1,
            order='first_line_index DESC',
        )
```